books/free-programming-books-cs.md

```diff
@@ -52,7 +52,7 @@
 
 ### Git
 
-* [Pro Git](https://knihy.nic.cz/#ProGit) - Scott Chacon (PDF, EPUB, MOBI)
+* [Pro Git](https://knihy.nic.cz/#ProGit) - Scott Chacon, `trl.:` Ondrej Filip (PDF, EPUB, MOBI)
```

Comment:
what does `trl.:` mean?

Reply:
it's a new annotation like `et al.` to give roles to creators.

An abbreviation meaning: translator

not handled yet by parser neither documented because was first adopted a few weeks ago. Look at #7034

other role abbreviations: